src/js/jsx/shared/Draggable.jsx

```diff
@@ -182,14 +166,24 @@ define(function (require, exports, module) {
              * @param {Event} event
              */
             _handleDragMove: function (event) {
+                var flux = this.getFlux();
+
                 if (!this.state.dragging) {
                     this.setState({
                         dragging: true
                     });
 
-                    this.getFlux().actions.draganddrop.registerDragging(this._getDragItems(getDragItem(this.props)));
+                    if (this.props.onDragStart) {
+                        this.props.onDragStart();
+                    }
+
+                    var dragItems = this.props.getDragItems(this);
+                    flux.store("draganddrop").startDrag(dragItems);
                 } else {
-                    this.getFlux().store("draganddrop").moveAndCheckBounds({ x: event.clientX, y: event.clientY });
+                    flux.store("draganddrop").updateDrag(this.props.zone, {
```

Comment:
I added a `isRequired` `propType` for `zone` to `Draggable` instead, since there isn't really a valid default here. See de1a254.